aries_cloudagent/messaging/schemas/routes.py

```diff
@@ -104,28 +109,27 @@ class SchemaIdMatchInfoSchema(OpenAPISchema):
     )
 
 
-class AutoEndorseOptionSchema(OpenAPISchema):
-    """Class for user to input whether to auto-endorse the transaction or not."""
+class CreateTransactioForEndorserOptionSchema(OpenAPISchema):
+    """Class for user to input whether to create a transaction for endorser or not."""
 
-    auto_endorse = fields.Boolean(
-        description="Auto-endorse Transaction",
+    create_transaction_for_endorser = fields.Boolean(
+        description="Create Transaction For Endorser's  signature",
         required=False,
     )
 
 
-class EndorserDIDOptionSchema(OpenAPISchema):
-    """Class for user to input the DID associated with the requested endorser."""
+class ConnIdMatchInfoSchema(OpenAPISchema):
+    """Path parameters and validators for request taking connection id."""
 
-    endorser_did = fields.Str(
-        description="Endorser DID",
-        required=False,
+    conn_id = fields.Str(
+        description="Connection identifier", required=False, example=UUIDFour.EXAMPLE
```

Comment:
Since we attach an Endorser Name to the connection, can we ask for the Endorser Name instead of the Connection ID?

Reply:
I think so, for this we would first need to create a functionality where - the agent enters a metadata value and gets a record in return.